cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -235,22 +235,22 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
 
         val info = analyzer.findManagedFiles(inputDir, distinctPackageManagers, repositoryConfiguration)
         if (info.managedFiles.isEmpty()) {
-            println("No project found.")
+            println("No definition files found.")
         } else {
             val filesPerManager = info.managedFiles.mapKeysTo(sortedMapOf()) { it.key.managerName }
             var count = 0
 
             filesPerManager.forEach { (manager, files) ->
                 count += files.size
-                println("Found ${files.size} $manager project(s) at:")
+                println("Found ${files.size} $manager definition files(s) at:")
```

Comment:
Already merged, but you might want to fix that typo in a separate PR: "file**~~s~~**(s)" (same below)

Reply:
Thanks, done: https://github.com/oss-review-toolkit/ort/pull/5088.